modules/textures/test/basis-loader.spec.js

```diff
@@ -105,6 +106,30 @@ test('BasisLoader#transcode to explicit format', async (t) => {
   t.end();
 });
 
+test('BasisLoader#auto-select a decoder format', async (t) => {
+  const images = await load(TEST_URL, BasisLoader, {
+    worker: true,
+    basis: {
+      format: 'astc-4x4',
+      decoderFormat: 'auto'
```

Comment:
Could this be `containerFormat`? (In the sense that we are dealing with basis encoded data inside a KTX 2 "container"?

Reply:
Done